src/tests/amiHelper.test.js

```diff
@@ -5,79 +5,90 @@
  */
 
 "use strict";
-const { EC2Client } = require("@aws-sdk/client-ec2");
 const Ami = require("../ami/AmiHelper.js");
 
-var ami;
-const amiName = "ami-jest-1";
+let ami;
+let amiName;
 
 beforeAll(() => {
     ami = new Ami("eu-west-3");
+    amiName = "team-backup-ami-jest-1";
 });
 
-test('AMICreate_InstanceExist_RecivedAnAMIID', async () => {
+test('exists_AmiExist_Success', async () => {
```

Comment:
Please meet this requirement https://nicolas-glassey.gitbook.io/vir1/notion-of-done/test-sample